Feature/use-post-endpoint

spences10 opened the pull request:
Use SvelteKit endpoints for posts reduces page load, like, lots!

genesisgabiola commented:
Fix #76

spences10 commented on content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx:
```diff
@@ -33,8 +33,9 @@ the past:
 
-This time round it was just as confusing (for me) at the times before,
-so this is why I'm making notes for future Scott to refer to. 🔥
+This time 'round it was just as confusing (for me) at the times
```
Remove this change, I don't use this punctuation

spences10 commented on content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx:
```diff
@@ -78,8 +79,8 @@ instructions from the [Windows WSL Install] docs.
 
-If you don't have Windows Pro you should be able to enable WSL 2 on
-[Windows 10 home].
+If you don't have Windows Pro you should still be able to enable WSL 2
```
Unnecessary word, remove this please

spences10 commented on content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx:
```diff
@@ -246,8 +247,8 @@ dev are VSCode and **WSLGit**.
-Prior to this I [had to install Git on Windows] which had it's own set
-of problems with line endings and file permissions.
+Prior to this, I [had to install Git on Windows] which had it's own
+set of problems with line endings and file permissions.
```
Unnecessary punctuation, remove this please

spences10 commented on content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx:
```diff
@@ -315,8 +316,7 @@ omf install nvm
 my [Cheat Sheets] site. This is shortening things like `git` to `g`
-(yes) and `yarn add` to `ya`. this time I'll need to move the created
-`.ssh` folder to
+(yes) and `yarn add` to `ya`.
```
Thanks for this, much appreciated, multi cursor fail!

spences10 commented on content/posts/2020/08/26/wsl-bootstrap-2020/index.mdx:
````diff
@@ -396,7 +396,7 @@ a try:
 
 ```bash
-sudo apt innstall unzip
+sudo apt install unzip
````
Thanks for this 🙏

spences10 reviewed (changes requested):
Thanks for this @genesisgabiola, please refer to my comments on parts I'm not willing to put in there and thanks for the typos.

spences10 commented:
Awesome! Thanks you so much @genesisgabiola 🙏

genesisgabiola commented:
> Awesome! Thanks you so much @genesisgabiola 🙏

I'm happy to contribute to your project 👍

genesisgabiola commented:
fix #76

spences10 committed:
feat: :sparkles: use endpoint for posts

spences10 committed:
feat: :sparkles: use endpoint for posts

spences10 committed:
feat: :sparkles: use endpoint for posts

renovate-bot committed:
chore(deps): update dependency svelte-lazy to v1.0.12

spences10 committed:
feat: :construction: add search filter

renovate-bot committed:
chore(deps): update dependency @sveltejs/adapter-vercel to v1.0.0-next.29

spences10 committed:
feat: :sparkles: sort endpoint and add filter component

spences10 committed:
refactor: :recycle: remove string values for height and width

spences10 committed:
feat: :sparkles: add slug.json endpoint

renovate-bot committed:
chore(deps): update dependency svelte-preprocess to v4.9.5

renovate-bot committed:
chore(deps): update dependency svelte-preprocess to v4.9.5

spences10 committed:
docs: :memo: add multiple fetch example

spences10 committed:
refactor: :recycle: use remarrk-preview in place of truncate-html

spences10 committed:
refactor: :recycle: use remarrk-preview in place of truncate-html

spences10 committed:
refactor: :recycle: use remarrk-preview in place of truncate-html

spences10 committed:
refactor: :recycle: use remarrk-preview in place of truncate-html

spences10 committed:
feat: :sparkles: add tags endpoint

spences10 committed:
feat: :sparkles: add slug endpoint for tags

spences10 committed:
feat: :sparkles: move slug file from folder

spences10 committed:
feat: :sparkles: use tags endpoint

spences10 committed:
refactor: :wrench: add preview

renovate-bot committed:
chore(deps): update dependency autoprefixer to v10.3.5

spences10 committed:
chore: :arrow_up:

spences10 committed:
feat: :sparkles: trim html for post card

spences10 committed:
chore: :bug: add preview back in

spences10 committed:
docs: :memo: update wsl win 11 post

renovate-bot committed:
chore(deps): update dependency eslint to v8.1.0

spences10 committed:
Merge pull request #88 from spences10/renovate/eslint-8.x

renovate-bot committed:
chore(deps): update dependency marked to v3.0.8

renovate-bot committed:
chore(deps): update dependency daisyui to v1.16.0

spences10 committed:
docs: :memo: update urql post

spences10 committed:
Merge pull request #89 from spences10/renovate/marked-3.x

spences10 committed:
Merge pull request #87 from spences10/renovate/daisyui-1.x

spences10 committed:
docs: :memo: wrap using URQL

spences10 committed:
Merge branch 'main' of github.com:spences10/scottspence.com into main

spences10 committed:
docs: :memo: build on URQL client and publish

renovate-bot committed:
chore(deps): update dependency autoprefixer to v10.4.0

renovate-bot committed:
chore(deps): update dependency graphql to v15.7.2

spences10 committed:
Merge pull request #91 from spences10/renovate/autoprefixer-10.x

spences10 committed:
Merge pull request #90 from spences10/renovate/graphql-15.x

renovate-bot committed:
chore(deps): update dependency graphql to v16

renovate-bot committed:
chore(deps): update dependency tailwindcss to v2.2.19

spences10 committed:
Merge pull request #92 from spences10/renovate/graphql-16.x

spences10 committed:
Merge pull request #93 from spences10/renovate/tailwindcss-2.x

spences10 committed:
refactor: :wrench: update vite fs allow locations

spences10 committed:
fix: :bug: fix rss meta data

renovate-bot committed:
chore(deps): update dependency svelte to v3.44.1

spences10 committed:
Merge pull request #94 from spences10/renovate/svelte-3.x

spences10 committed:
refactor: :recycle: add aliases

spences10 committed:
chore: :arrow_up: update daisyui and graphql

spences10 committed:
style: :lipstick: remove unused styles

spences10 committed:
refactor: :recycle: config file paths for vite

renovate-bot committed:
chore(deps): update dependency cssnano to v5.0.9

renovate-bot committed:
chore(deps): update dependency marked to v4

spences10 committed:
docs: :memo: add Ubuntu backup

spences10 committed:
docs: :memo: backslashes

spences10 committed:
chore: :wrench: update lockfile

spences10 committed:
chore: :wrench: enable file types

spences10 committed:
chore: :recycle: move to index and folder

spences10 committed:
docs: :memo: add listed instances

spences10 committed:
docs: :memo: remove demo link, update dashboard link

spences10 committed:
docs: :memo: Start Fathom Analytics with SvelteKit post

spences10 committed:
docs: :memo: spellings mistakes

spences10 committed:
fix: :bug: fix feed metadata export

spences10 committed:
chore: :wrench: update peeps met

spences10 committed:
Merge pull request #97 from spences10/renovate/cssnano-5.x

spences10 committed:
docs: :memo: update fathom post

spences10 committed:
Merge branch 'main' of github.com:spences10/scottspence.com into main

spences10 committed:
fix: :bug: use named import

spences10 committed:
Merge pull request #98 from spences10/renovate/marked-4.x

spences10 committed:
docs: :memo: update faq

spences10 committed:
chore: :wrench: update popular posts stats

spences10 committed:
style: :lipstick: add link hover

spences10 committed:
fix: :bug: roll back marked version

spences10 committed:
style: :lipstick: remove border use shadow

spences10 committed:
refactor: :recycle: add peeps to list

spences10 committed:
docs: :memo: add fastmail config

spences10 committed:
refactor: :recycle: remove prerender

spences10 committed:
Merge branch 'main' of github.com:spences10/scottspence.com into main

spences10 committed:
docs: update fathom post

spences10 committed:
update marked

renovate-bot committed:
chore(deps): update dependency prettier-plugin-svelte to v2.5.0

spences10 committed:
Merge pull request #102 from spences10/renovate/prettier-plugin-svelte-2.x

spences10 committed:
docs: :memo: document getting started with Twilwind in Svelte

spences10 committed:
Merge branch 'main' of github.com:spences10/scottspence.com into main

renovate-bot committed:
chore(deps): update dependency cssnano to v5.0.11

spences10 committed:
docs: :memo: finish up tailwind and svelte post

spences10 committed:
docs: :memo: update meeting link

spences10 committed:
Merge pull request #104 from spences10/renovate/cssnano-5.x

renovate-bot committed:
chore(deps): update dependency date-fns to v2.26.0

renovate-bot committed:
chore(deps): update dependency svelte to v3.44.2

spences10 committed:
Merge pull request #105 from spences10/renovate/date-fns-2.x

spences10 committed:
docs: :memo: typo!

spences10 committed:
Merge branch 'main' of github.com:spences10/scottspence.com into main

spences10 committed:
chore: :memo: tabs to spaces

spences10 committed:
Merge pull request #106 from spences10/renovate/svelte-3.x

spences10 committed:
typo and spaces for tabs

renovate-bot committed:
chore(deps): update dependency marked to v4.0.4

renovate-bot committed:
chore(deps): update dependency eslint to v8.3.0

renovate-bot committed:
chore(deps): update dependency slugify to v1.6.3

spences10 committed:
Merge pull request #107 from spences10/renovate/marked-4.x

spences10 committed:
Merge pull request #108 from spences10/renovate/eslint-8.x

spences10 committed:
Merge pull request #109 from spences10/renovate/slugify-1.x

spences10 committed:
docs: :memo: add prereq and spaces for tabs

spences10 committed:
Merge branch 'main' of github.com:spences10/scottspence.com into main

spences10 committed:
chore: :wrench: update lockfile

spences10 committed:
refactor: :recycle: aria label for scroll down for more

spences10 committed:
chore: :recycle: fenangle deps

spences10 committed:
refactor: :recycle: remove prerender flag on routes

renovate-bot committed:
chore(deps): update dependency marked to v4.0.5

renovate-bot committed:
chore(deps): update dependency prettier to v2.5.0

renovate-bot committed:
chore(deps): update dependency prettier to v2.5.0

spences10 committed:
Merge pull request #111 from spences10/renovate/marked-4.x

renovate-bot committed:
chore(deps): update dependency cssnano to v5.0.12

renovate-bot committed:
chore(deps): update dependency cssnano to v5.0.12

spences10 committed:
chore: :memo: Smartypants cleanup

spences10 committed:
docs: :memo: data fetching notes

spences10 committed:
chore: :memo: meeting link

spences10 committed:
chore: :wrench: update spellings

spences10 committed:
chore: :bento: add assets for places coded

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 335.21kb -> 287.18kb (14.33%)

/posts/2021/11/16/how-to-set-up-svelte-with-tailwind/changes-after-svelte-add-tailwind.png -- 15.74kb -> 9.45kb (39.97%)
/src/lib/icons/asi.svg -- 131.79kb -> 91.12kb (30.86%)
/src/lib/icons/bt.svg -- 1.12kb -> 1.03kb (7.79%)
/src/lib/icons/astrazeneca.svg -- 5.40kb -> 5.13kb (5.13%)
/src/lib/icons/honda.svg -- 6.21kb -> 6.16kb (0.85%)
/src/lib/icons/ncsc.svg -- 99.11kb -> 98.50kb (0.61%)
/posts/2021/02/01/logitech-g915/chonka-cable.jpg -- 75.84kb -> 75.80kb (0.05%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

spences10 committed:
chore: :recycle: change to svelte components

spences10 committed:
chore: :recycle: change to svelte components

spences10 committed:
feat: :recycle: import componants

hate how it looks and decide not to use them

spences10 committed:
feat: :recycle: import componants

hate how it looks and decide not to use them

spences10 committed:
chore: :recycle: get rid of svgs

spences10 committed:
chore: :recycle: get rid of svgs

spences10 committed:
Merge pull request #114 from spences10/renovate/cssnano-5.x

spences10 committed:
Merge pull request #112 from spences10/renovate/prettier-2.x

renovate-bot committed:
chore(deps): update dependency postcss to v8.4.4

renovate-bot committed:
chore(deps): update dependency postcss to v8.4.4

renovate-bot committed:
chore(deps): update dependency date-fns to v2.27.0

renovate-bot committed:
chore(deps): update dependency date-fns to v2.27.0

renovate-bot committed:
chore(deps): update dependency marked to v4.0.6

renovate-bot committed:
chore(deps): update dependency marked to v4.0.6

spences10 committed:
docs: :memo: add use fetch with variables in SvelteKit

spences10 committed:
docs: :memo: add use fetch with variables in SvelteKit

spences10 committed:
Merge pull request #110 from spences10/renovate/postcss-8.x

spences10 committed:
Merge pull request #116 from spences10/renovate/date-fns-2.x

renovate-bot committed:
chore(deps): update dependency eslint to v8.4.0

renovate-bot committed:
chore(deps): update dependency eslint to v8.4.0

spences10 committed:
Merge pull request #117 from spences10/renovate/marked-4.x

renovate-bot committed:
chore(deps): update dependency graphql-request to v3.7.0

renovate-bot committed:
chore(deps): update dependency prettier to v2.5.1

renovate-bot committed:
chore(deps): update dependency prettier to v2.5.1

spences10 committed:
Merge pull request #119 from spences10/renovate/eslint-8.x

spences10 committed:
Merge pull request #120 from spences10/renovate/prettier-2.x

renovate-bot committed:
chore(deps): update dependency eslint to v8.4.1

spences10 committed:
fix: :bug: update hours

spences10 committed:
fix: :bug: update hours

spences10 committed:
docs: :memo: update speaking

spences10 committed:
docs: :memo: update speaking

spences10 committed:
Merge pull request #115 from spences10/imgbot

spences10 committed:
Merge pull request #118 from spences10/renovate/graphql-request-3.x

renovate-bot committed:
chore(deps): update dependency graphql to v16.1.0

spences10 committed:
building

spences10 committed:
docs: :memo: typo party

renovate-bot committed:
chore(deps): update dependency prettier-plugin-svelte to v2.5.1

spences10 committed:
docs: :memo: powershell backslashes

spences10 committed:
docs: :memo: add fedora 35 bootstrap guide

spences10 committed:
Merge pull request #121 from spences10/renovate/eslint-8.x

spences10 committed:
Merge pull request #122 from spences10/renovate/graphql-16.x

renovate-bot committed:
chore(deps): update dependency marked to v4.0.7

spences10 committed:
Merge pull request #123 from spences10/renovate/prettier-plugin-svelte-2.x

renovate-bot committed:
chore(deps): update dependency postcss to v8.4.5

renovate-bot committed:
chore(deps): update dependency @sveltejs/adapter-vercel to v1.0.0-next.32

spences10 committed:
Merge pull request #126 from spences10/renovate/marked-4.x

spences10 committed:
Merge pull request #128 from spences10/renovate/postcss-8.x

spences10 committed:
Merge pull request #130 from spences10/renovate/sveltejs-adapter-vercel-1.x

spences10 committed:
chore: :wrench: update lockfile

renovate-bot committed:
chore(deps): update dependency tailwindcss to v3

spences10 committed:
chore: :arrow_up: update deps

spences10 committed:
Merge pull request #125 from spences10/renovate/tailwindcss-3.x

renovate-bot committed:
chore(deps): update dependency svelte to v3.44.3

renovate-bot committed:
chore(deps): update dependency daisyui to v1.16.6

spences10 committed:
Merge pull request #127 from spences10/renovate/daisyui-1.x

renovate-bot committed:
chore(deps): update dependency graphql to v16.2.0

spences10 committed:
Merge pull request #131 from spences10/renovate/svelte-3.x

renovate-bot committed:
chore(deps): update dependency slugify to v1.6.4

renovate-bot committed:
chore(deps): update dependency tailwindcss to v3.0.7

renovate-bot committed:
chore(deps): update dependency eslint to v8.5.0

renovate-bot committed:
chore(deps): update dependency svelte-preprocess to v4.10.1

spences10 committed:
style: :lipstick: add victor mono

spences10 committed:
Merge pull request #135 from spences10/renovate/graphql-16.x

renovate-bot committed:
chore(deps): update dependency marked to v4.0.8

spences10 committed:
Merge branch 'main' of github.com:spences10/scottspence.com into main

spences10 committed:
style: :lipstick: Update people to meet

Check off paulie, add rel="noreferrer noopener" target="_blank"

spences10 committed:
feat: :sparkles: use Tailwind typography classes

update daisyUI

spences10 committed:
Merge pull request #134 from spences10/renovate/tailwindcss-3.x

spences10 committed:
Merge pull request #136 from spences10/renovate/eslint-8.x

spences10 committed:
Merge pull request #133 from spences10/renovate/slugify-1.x

spences10 committed:
Merge pull request #129 from spences10/renovate/svelte-preprocess-4.x

renovate-bot committed:
chore(deps): update dependency cssnano to v5.0.14

spences10 committed:
Merge pull request #137 from spences10/renovate/marked-4.x

renovate-bot committed:
chore(deps): update dependency fuse.js to v6.5.3

spences10 committed:
Merge pull request #132 from spences10/renovate/cssnano-5.x

renovate-bot committed:
chore(deps): update dependency daisyui to v1.20.0

spences10 committed:
Merge branch 'main' of github.com:spences10/scottspence.com into main

spences10 committed:
refactor: :recycle: add views to popular posts on footer

spences10 committed:
style: :bug: h-full instead of h-max

spences10 committed:
feat: :sparkles: default to valentine

spences10 committed:
feat: :sparkles: add buy me a coffee component

spences10 committed:
docs: :memo: start Keychron Q1 post

spences10 committed:
docs: :memo: lot's to choose from

keycaps, switches, more headings to follow up on

spences10 committed:
docs: :memo: publish keychron post

spences10 committed:
typo PCB

spences10 committed:
Merge pull request #139 from spences10/renovate/fuse.js-6.x

spences10 committed:
Merge pull request #138 from spences10/renovate/daisyui-1.x

spences10 committed:
chore: :arrow_up: update deps

spences10 committed:
chore: :arrow_up: update deps

spences10 committed:
refactor: :recycle: replace page with params

breaking change in requst.origin/path/query, see https://github.com/sveltejs/kit/pull/3126

all places where page is used in a load function

spences10 committed:
chore: :arrow_up: update lockfile

spences10 committed:
chore: :arrow_up: upgrade vercel adapter

renovate-bot committed:
chore(deps): update dependency eslint to v8.6.0

spences10 committed:
feat: :sparkles: break up speaking into years

spences10 committed:
chore: :wrench: update lockfile

spences10 committed:
feat: :sparkles: select year

spences10 committed:
feat: :sparkles: add content for 22

spences10 committed:
refactor: :recycle: extra margin

spences10 committed:
Merge pull request #149 from spences10/feat/speaking-breakdown

spences10 committed:
feat: :sparkles: use cloudinary for images

spences10 committed:
refactor: :fire: remove assets

spences10 committed:
refactor: :recycle: use cloudinary link

spences10 committed:
refactor: :recycle: add cloudinary asset link

spences10 committed:
Merge pull request #151 from spences10/cloudinary-images

spences10 committed:
Merge pull request #147 from spences10/renovate/eslint-8.x

renovate-bot committed:
chore(deps): update dependency rehype-slug to v5.0.1

renovate-bot committed:
chore(deps): update dependency date-fns to v2.28.0

renovate-bot committed:
chore(deps): update dependency autoprefixer to v10.4.1

renovate-bot committed:
chore(deps): update dependency postcss-load-config to v3.1.1

spences10 committed:
Merge pull request #140 from spences10/renovate/date-fns-2.x

spences10 committed:
Merge pull request #146 from spences10/renovate/rehype-slug-5.x

spences10 committed:
Merge pull request #142 from spences10/renovate/autoprefixer-10.x

renovate-bot committed:
chore(deps): update dependency rehype-autolink-headings to v6.1.1

spences10 committed:
Merge pull request #144 from spences10/renovate/rehype-autolink-headings-6.x

spences10 committed:
feat: :sparkles: add skip to navigation tag

spences10 committed:
fix: :lipstick: fix mobile padding

renovate-bot committed:
chore(deps): update dependency daisyui to v1.21.0

renovate-bot committed:
chore(deps): update dependency slugify to v1.6.5

spences10 committed:
refactor: :recycle: add Jay

spences10 committed:
refactor: :recycle: add GHOSTY!!!!

spences10 committed:
Merge pull request #150 from spences10/renovate/postcss-load-config-3.x

spences10 committed:
Merge pull request #145 from spences10/renovate/daisyui-1.x

spences10 committed:
Merge pull request #152 from spences10/renovate/slugify-1.x

spences10 committed:
style: :wrench: remove typography max width

spences10 committed:
style: :lipstick: remove css variables for background clip

spences10 committed:
refactor: :lipstick: change gradient to bottom

spences10 committed:
revert!

spences10 committed:
revert!

spences10 committed:
cleanup!
